### PR TITLE
code for check

### PR DESCRIPTION
**Pull request by kristinagerdt**
Issues: #1 #2 #3 #4 #5

### CODE REVIEW

File: src/IsCircular.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            }
+        }
+        return false;
+    }
+
+    public static boolean isCircular2(LinkedList<String> list1, LinkedList<String> list2) {
+        for (int i = 0; i < list1.size(); i++) {
+            list1.addLast(list1.getFirst());
+            list1.removeFirst();
+            if (list1.equals(list2)) {
```
Comment (sergeylukichev) [inline src/IsCircular.java]: I think there is a method in linked list which returns you the first element and removes it from its position in the list. You may save one call.

File: src/Reverse.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import java.util.*;
+
```
Comment (sergeylukichev) [inline src/Reverse.java]: It is a good practice to import only classes you actually use, do not use a wildcard.

**Comment by sergeylukichev** [comment]
👍

File: src/IsCircular.java (hunk 1 above)
Comment (kristinagerdt) [reply src/IsCircular.java]: fixed

File: src/Reverse.java (hunk 2 above)
Comment (kristinagerdt) [reply src/Reverse.java]: fixed

**Pull request merged**